kokoro/scripts/test/start_soak_test.sh

```diff
@@ -5,6 +5,23 @@ set -u
 set -x
 set -o pipefail
 
+# cd to the root of the git repo containing this script ($0).
+cd "$(readlink -f "$(dirname "$0")")"
+cd ../../../
+
+# Source the common utilities.
+source kokoro/scripts/utils/louhi.sh
+
+# For soak tests run by Louhi
+read_louhi_tag
+# if TARGET & ARCH are set, retrieve the soak distro from project.yaml
+if [[ -n "${TARGET:-}" && -n "${ARCH:-}" ]]; then
+  DISTRO=$(yaml project.yaml "['targets']['${TARGET}']['architectures']['${ARCH}']['soak_distro']")
+  VM_NAME="soak-test-${REPO_SUFFIX}-${LABEL}"
+  export DISTRO
+  export VM_NAME
```

Comment:
Unlike `DISTRO`, this variable is not assigned via a process substitution, so it can be exported in the same statement. If you want to keep the exports separate for symmetry, you can export both variables in one statement.